update firewall update request object

ariquinones opened the pull request.

JoMas971 commented:
Code coverage goes down...
In that case you have 2 solutions :
- create tests for this file to avoid coverage to go down,
- use cory ts generator IntelliJ plugin to generate the ts class from JAVA response class.
It will include `/* istanbul ignore next: autogenerated */` and we won't need to test it SDK side, since it has been tested JAVA side (the ts class should be a perfect reflection of what we have Java side except in some cases...).

ariquinones commented:
@JoMas971 the request object I updated as well as the method used still have the /* istanbul ignore next: autogenerated */ comment so I'm wondering what file is bringing the coverage down? By updating the rules, I updated the other firewall.test.unit file and it tests all methods within this class, so what file or class do i need to create the test for so coverage doesn't go down?

JoMas971 commented on src/sdk/model/edge/firewall/edge-firewall-update-request.ts:
```diff
@@ -10,70 +11,81 @@ export class EdgeFirewallUpdateRequest {
-   * Get the json representation of this class.
-   * @returns {EdgeFirewallUpdateRequestJson}
-   */
+    /**
```
Wrong indentation in this file.

JoMas971 commented on src/sdk/model/edge/firewall/edge-firewall-rule-update-request.ts:
```diff
@@ -0,0 +1,194 @@
+     * Get description.
+     * @returns {string}
+     */
+  get description(): string {
```
Wrong indentation in this file.

JoMas971 commented:
I just have 2 little comments that need to be fixed. Otherwise, the code looks good. The coverage goes dow by only 0.02% which is negligible and caused by the new file addition. It consider it as a change when it shouldn't...
